content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -36,11 +36,86 @@ are supported. If you are on an older version, please upgrade to 1.1 first.
 [Download the new Istio release](/docs/setup/kubernetes/#downloading-the-release)
 and change directory to the new release directory.
 
+### Istio CNI upgrade
+
+Choose one of the following **mutually exclusive** options to check whether
```

Comment:
should we say if you are using CNI, then... we need to make sure it is noted as optional component thus this step is optional and only needed for some folks.

Reply:
note it is also possible user was not using CNI in 1.1 but want to switch to use CNI in 1.2

Reply:
Done.